website/docs/docs/build/documentation.md

```diff
@@ -128,9 +128,11 @@ models:
 
 In the resulting documentation, `'{{ doc("table_events") }}'` will be expanded to the markdown defined in the `table_events` docs block.
 
+
 ## Setting a custom overview
+*Currently available for the legacy dbt Docs only.*
```

Comment:
Should we be referring to it as `legacy` since it's still the docs experience for Core users? They'll be reading this page too, right? Maybe the "Previous dbt Cloud docs experience" or something similar?